### PR TITLE
Add system requirements to documentation

### PR DESCRIPTION
**Pull request by astrojuanlu**
## Description

Add system requirements to documentation to make them more explicit.

## Checklist

- [x] The documentation reflects the changes.
- [x] The PR branch is set up to merge into `dev` instead of `master`.
- [x] I haven't introduced breaking changes that would disrupt existing jobs, i.e. backwards compatibility is maintained.
- [x] In case I changed the dependencies in any `requirements.in` I have run `pip-compile` to update the corresponding `requirements.txt`.
- [x] In case I changed one of the services' `models.py` I have performed the appropriate database migrations (refer to the [DB migration docs](https://docs.orchest.io/en/stable/development/development_workflow.html#database-schema-migrations)).
- [x] In case I changed code in the `orchest-sdk` I followed its [release checklist](https://github.com/orchest/orchest/blob/master/orchest-sdk/python/RELEASE-CHECKLIST.md).
- [x] In case I changed code in the `orchest-cli` I followed its [release checklist](https://github.com/orchest/orchest/blob/master/orchest-cli/RELEASE-CHECKLIST.md).

### CODE REVIEW

**Comment by astrojuanlu** [comment]
Rendered docs: https://orchest--1051.org.readthedocs.build/en/1051/getting_started/installation.html#prerequisites

File: docs/source/getting_started/installation.rst [hunk 1]
```diff
@@ -24,6 +24,12 @@ locally, e.g. using `minikube
 
 Do make sure that, no matter the cluster you choose, the ingress controller is configured.
 
+The supported operating systems are:
+
+- Linux (``x86_64``)
+- `Windows Subsystem for Linux 2 <https://docs.microsoft.com/en-us/windows/wsl/about>`_ (WSL2)
+- macOS (M1 Macs use Rosetta for emulation)
```
Comment (yannickperrenet) [inline docs/source/getting_started/installation.rst:31]:
```suggestion
- macOS (M1 Macs should use Rosetta for emulation)
```

And maybe linking Rosetta to either of:

* https://osxdaily.com/2020/12/04/how-install-rosetta-2-apple-silicon-mac/
* https://support.apple.com/en-us/HT211861

I prefer the former as I am not sure whether running minikube or the likes actually gives that pop-up.

Comment (astrojuanlu) [reply docs/source/getting_started/installation.rst]: Good point! Done

**Review by yannickperrenet — APPROVED** [review]
I love having quality docs. Thanks for improving the quality :wink:

**Pull request merged**